node/sc/api_bridge.go

```diff
@@ -95,6 +95,14 @@ func (sb *SubBridgeAPI) GetAnchoringTxHashByBlockNumber(bn uint64) common.Hash {
 	return receipt.TxHash
 }
 
+func (sb *SubBridgeAPI) RegisterOperator(bridgeAddr, operatorAddr common.Address) (common.Hash, error) {
+	return sb.subBridge.bridgeManager.RegisterOperator(bridgeAddr, operatorAddr)
+}
+
+func (sb *SubBridgeAPI) GetRegisteredOperators(bridgeAddr common.Address) ([]common.Address, error) {
```

Comment:
```suggestion
func (sb *SubBridgeAPI) GetOperators(bridgeAddr common.Address) ([]common.Address, error) {
```

Reply:
Operators API already exists. To distinguish them, I named it.